src/Symfony/Component/Translation/Loader/XliffFileLoader.php

```diff
@@ -48,14 +48,14 @@ public function load($resource, $locale, $domain = 'messages')
         foreach ($xml->xpath('//xliff:trans-unit') as $translation) {
             $attributes = $translation->attributes();
 
-            if (!(isset($attributes['resname']) || isset($translation->source)) || !isset($translation->target)) {
+            if (!(isset($attributes['resname']) || isset($translation->source))) {
```

Comment:
why do we also skip when resname is not set? It's an optional attribute according to the specs